nodes.lua

```diff
@@ -459,6 +459,12 @@ for name, def in pairs(nodes) do
 	def.tiles = def.tiles or {"moreblocks_" ..name.. ".png"}
 	minetest.register_node("moreblocks:" ..name, def)
 	minetest.register_alias(name, "moreblocks:" ..name)
+
+	-- Use the primary tile for all sides of cut glasslike nodes
+	if #def.tiles > 1 and def.drawtype and def.drawtype:find("glass") then
```

Comment:
`(def.drawtype == "glasslike_framed" or def.drawtype == "glasslike_framed_optional")` to avoid false positives